common/tests/test_cryptoWrapper.py

```diff
@@ -204,7 +204,7 @@
   exit(-1)
 
 c = list(ciphertext)
-c[0] = c[0] + 1
+c[0] = (c[0] + 1) % 256 # Make sure it stays a byte or swig might fail find the correct C++ function below
 ciphertext = tuple(c)
```

Comment:
Just curious about why the + 1 was put in the first place. (often done to avoid 0 in the range, but then after %256, you allow 0)

Reply:
this is a test-program where the +1 (intentionally) tampers with the IV to cause a ciphertext which should not decrypt.  The problem now was that if the first byte of the IV happened to be 255, the +1 in python promoted it to a larger integer-type which then caused swig not to find a correct C++ implementation of the crypto functionality ...